apps/modernization-ui/src/pages/addPatient/components/contactFields/ContactFields.tsx

```diff
@@ -144,7 +144,7 @@ export default function ContactFields({ id, title }: Props) {
                             }
                             className="text-bold"
                             unstyled
-                            style={{ margin: '0', padding: '0' }}>
+                            style={{ margin: '10px 0 0 0', padding: '0' }}>
```

Comment:
issue:  This gap should be set to 1.125rem.  Also, this is a great candidate for a refactor.  All of the elements within the card should have a gap of 1.125rem, I think the use of `Grid` is complicating the styles.

Reply:
okay will look into it thanks